tracing/src/main/java/com/palantir/tracing/AsyncSlf4jSpanObserver.java

```diff
@@ -45,7 +44,6 @@
 public final class AsyncSlf4jSpanObserver extends AsyncSpanObserver {
```

Comment:
interestingly I actually can't find any references to this class internally... a separate PR could even do a cheeky deprecation/delete??

Reply:
Yep, as noted on the pr description ;-)

Reply:
I don think it’s helpful to provide an oss reference implementation of some kind though, so my preference is the external module